Public/Src/Engine/Dll/Tracing/Log.cs

```diff
@@ -1868,6 +1868,26 @@ public void BusyOrUnavailableOutputDirectories(LoggingContext context, string ob
             Message = "/unsafe_DisableSharedOpaqueEmptyDirectoryScrubbing: removal of empty directories within shared opaques has been disabled. This is an unsafe configuration since it may work in detriment of build correctness.")]
         public abstract void ConfigUnsafeDisableSharedOpaqueEmptyDirectoryScrubbing(LoggingContext context);
 
+        [GeneratedEvent(
+            (int)EventId.ScrubbingOutputsFromJournalStarted,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Informational,
+            Keywords = (int)(Keywords.UserMessage | Keywords.Overwritable),
+            EventTask = (int)Tasks.Engine,
+            EventOpcode = (byte)EventOpcode.Start,
+            Message = EventConstants.PhasePrefix + "Scrubbing shared opaque outputs explicitly recorded in the journal.")]

```

Comment:
>Scrubbing [](start = 52, length = 9)

I'd love that we eventually switch from 'scrub' to 'delete' as the term we expose to the end user when deleting shared opaque outputs.
I think it is just an implementation detail that we use the same 'scrubber' class for both the /scrub option and for deleting old shared opaque outputs. Bxl uses 'scrub' to mean 'remove extraneous files', but what we actually do for shared opaque outputs is just a deletion.
(and I know I'm personally to blame for leaking that term to being with :( )

Reply:
I'll change it here to delete